app/src/main/java/com/concordia/mcga/adapters/DirectionsArrayAdapter.java

```diff
@@ -0,0 +1,104 @@
+package com.concordia.mcga.adapters;
+
+
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.concordia.mcga.activities.R;
+
+import java.util.ArrayList;
+
+import static android.content.ContentValues.TAG;
+
+
+public class DirectionsArrayAdapter extends ArrayAdapter<String> {
+    private final Context context;
+    private ArrayList<String> directionsText = new ArrayList<String>();
```

Comment:
I agree with @adriannadiaz, it's still good practice when writing java to use the interface because then ArrayList can be safely replaced if ever it needs to be.